README.md

```diff
@@ -117,9 +117,3 @@ To add new files or dependencies to this exported package, have a look at [scrip
 If you have any questions, ping us on Slack
 (https://slack.opencollective.com) or on Twitter
 ([@opencollect](https://twitter.com/opencollect)).
-
-## Thanks
-
-<a href="https://www.chromatic.com/"><img src="https://user-images.githubusercontent.com/321738/84662277-e3db4f80-af1b-11ea-88f5-91d67a5e59f6.png" width="153" height="30" alt="Chromatic" /></a>
-
-Thanks to [Chromatic](https://www.chromatic.com/) for providing the visual testing platform that helps us review UI changes and catch visual regressions.
```

Comment:
Maybe we can actually replace that with a thanks + link to Lost Pixel?

Reply:
Done and yes, I was going to add lost pixel badge as suggested (https://github.com/lost-pixel/lost-pixel#using-lost-pixel-platfrom-in-non-commerical-open-source-projects), but I think this is a better idea. 😉